docs/en/observability/monitor-infra/kubernetes-pod-metrics.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[kubernetes-pod-metrics]]
+= Kubernetes pod metrics
+
+//TODO: Rewrite this to match new format. Requires input from dev team.
+
+//REVIEWERS: This was taken from the existing docs and moved here. Do we want
```

Comment:
Reviewers: Please respond to this question about pod metrics. What do we want to do with the existing docs about non-host metrics? Do we want to try to update them, or just leave as-is for now?